http/file_server_test.ts

```diff
@@ -989,7 +991,7 @@ Deno.test(
 );
 
 Deno.test("file_server prints local and network urls", async () => {
-  const port = await getAvailablePort();
+  const port = 4507;
```

Comment:
Actually, you're right. I think instead of removing the functions, we should advise against using it, pointing to `port: 0` in almost all cases. This is one of the edge use cases. WDYT?

Reply:
Documeting and recommending `port: 0` in `getAvailablePort` docs sounds good to me.